test/utilities.ts

```diff
@@ -53,9 +53,10 @@ export async function closeAuction(
 }
 
 export async function sendTxAndGetReturnValue<T>(
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   method: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     sendTransaction: (...args: any[]) => Promise<string>;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
nit: This shouldn't be needed anymore if merging with master:
https://github.com/gnosis/dex-contracts/blob/master/test/utilities.ts#L55-L67

Reply:
Awesome thanks. Was that recent? I just branch off master this morning (or so I thought)

Reply:
It was from this morning.